### ISSUE TITLE
&FULLPATH Bug

### ISSUE THREAD

**Issue by zAOkx-sft** [halcyon-tech/vscode-ibmi#845]
Type: <b>Bug</b>

&FULLPATH is not working like it used to. Before, when I was in the IBM i IFS Broswer, after I deployed some files into there, when I used a compile command that had &FULLPATH in it, it would automatically pick up my workspce location file path from inside of the IFS browser. Now, it is doing some strange combination of the beginning of the IFS directory mixed with my local machine C drive. 

Extension version: 1.5.4
VS Code version: Code 1.70.2 (e4503b30fc78200f846c62cf8091b76ff5547662, 2022-08-16T05:35:13.448Z)
OS version: Windows_NT x64 10.0.19044
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz (12 x 2712)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: unavailable_off<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.73GB (21.73GB free)|
|Process Argv|--crash-reporter-id 0f847544-386b-4e1d-9648-15ac52ec2fc4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30557514
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
pylanb8912:30545647
vsclangdc:30486549
c4g48928:30535728
hb751961:30553087
dsvsc012cf:30540253
azure-dev_surveyonecf:30548226
2144e591:30553903

```

</details>

**Comment by worksofliam** [comment]
Yup. I will fix it back to be the IFS path of the uploaded file.

**Comment by worksofliam** [comment]
@zAOkx-sft About to make a release.

**Comment by worksofliam** [comment]
1.5.10 is on the way! I will close this but please do open if you find another issue.

**Comment by zAOkx-sft** [comment]
> 1.5.10 is on the way! I will close this but please do open if you find another issue.

Thanks so much for the help! I will let you know if there is another issue. 

Have a great day!
